### DISCUSSION TITLE
Bug: Unpausing all players simultaneously does not work after adjusting a singe player

### DISCUSSION THREAD

**Discussion by Toastie** [f1multiviewer/issue-tracker#120] [General]
Steps to reproduce:

1. Open multiple players of a replay
2. Pause all together with `shift + space`
3. Unpause a single player with either `space` or the play/pause button
4. Pause this player again, all players should be paused now, the focus stays on the now again paused player
5. Unpause all players with `shift + space` again

Expected behaviour at this point:
All players start playing again.

Actual behaviour:
All players execpt the manually adjusted player start playing immediately and continue to do so.
The player that has been adjusted starts playing and after that immediately pauses again.

Known workarounds:
After step 4, exiting fullscreen on the adjusted player and entering again causes the expected behaviour to occur after step 5.

Known affected versions:
1.4.0. and 1.4.4.

**Comment by jeppeolesen** [comment]
Hey there @Toastie 

If this is still an issue, please submit a bug report over here: https://github.com/f1multiviewer/issue-tracker/issues

**Reply by Toastie** [reply]
Huh funny timing, havent had any issues with pausing replays until literally just now. Shitft + Space doesn't work at all for me, was perfectly fine last weeked. I'll create an issue after I'm done watching the race.

**Reply by LapsTimeOFF** [reply]
Thank you 🫶
